### DISCUSSION TITLE
Strange created filenames.

### DISCUSSION THREAD

**Discussion by tblock-zz** [stitionai/devika#564] [Q&A]
I am using devika on an ubuntu 22.04 LTS machine with a local running ollama with various models.
In principle it is creating files, but the file names are strange like: 
tom@tomHome test2 $ ls
'`main.py`'
Where can I fix that, so the filename would be main.py?

**Comment by kgott** [comment]
I saw this too. I think changing line 55 in src/agents/coder.py from:

```
current_file = line.split(":")[1].strip()
```

to:

```
current_file = line.split(":")[1].replace("`"", "").strip()
```

would fix this. It'a actually the backquote character, not a single quote. It looks like the backquotes are meant to print the name of the generated file in the terminal window in a monospace font.

**Comment by kgott** [comment]
I opened issue #566 for this and created PR #567 to fix this issue using the change that I mentioned in my previous comment.
